packages/ember-toucan-core/src/components/button.hbs

```diff
@@ -9,7 +9,7 @@
     {{yield to="loading"}}
     <span class="sr-only" data-loading>{{yield}}</span>
   {{else if @isDisabled}}
-    <span class="flex items-center gap-x-2">
+    <span class="flex flex-grow items-center gap-x-2">
```

Comment:
Based on your description, does the `flex-grow` only need to be here when the button is disabled?

Reply:
that's where I've encountered the error